instance/tests/test_tasks.py

```diff
@@ -369,8 +369,8 @@ def test_shut_down_obsolete_pr_sandboxes(self, data, mock_archive, mock_logger):
             closed_at = None
 
         with patch(
-            'pr_watch.github.get_pr_info_by_number',
-            return_value={'state': pr_state, 'closed_at': closed_at},
+                'pr_watch.github.get_pr_info_by_number',
```

Comment:
Is it `pylint` which ask for double spaced tabs under `with` statement?

Reply:
Yes